ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,36 @@
+﻿.govuk-file-upload {
+    font-size: 1rem;
+    outline: 3px solid transparent;
+    border-color: transparent;
+    box-shadow: inset 0 0 0 2px transparent;
+    padding: 3px 3px 1px 3px;
```

Comment:
This padding can be removed. It actually causes the button to be misaligned relative to other buttons on the page.

<img width="268" alt="buttons not aligned" src="https://github.com/user-attachments/assets/a66084f4-5b91-4cdb-a4e6-0caa147a9677">

Reply:
If the padding is removed, there is no gap between the shadow and the button
With padding 
![image](https://github.com/user-attachments/assets/5d600462-1a18-46c8-b637-c5783076a202)
Without padding
![image](https://github.com/user-attachments/assets/876722f5-affa-4cc7-bfc7-daec72fa1b2e)